addons/web/static/lib/pdfjs/web/ui_utils.js

```diff
@@ -348,6 +348,9 @@ var ProgressBar = (function ProgressBarClosure() {
       this.div.classList.remove('indeterminate');
       var progressSize = this.width * this._percent / 100;
       this.div.style.width = progressSize + this.units;
+      this.bar.setAttribute('aria-valuemin', 0);
+      this.bar.setAttribute('aria-valuemax', 100);
+      this.bar.setAttribute('aria-valuenow', this._percent);
```

Comment:
I was able to deduce what this does, but I'd add a comment saying "this makes the progress bar visible to screen readers"